### PR TITLE
adding instructions to disable alert forwarding for managed clusters

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/stolostron/backlog/issues/26981

### CODE REVIEW

File: observability/customize_observability.adoc [hunk 1]
```diff
@@ -5,25 +5,25 @@ Review the following sections to learn more about customizing, managing, and vie
...
+- <<adding-custom-metrics, Adding custom metrics>>
+- <<removing-default-metrics,Removing default metrics>>
+- <<export-metrics-to-external-endpoints,Export metrics to external endpoints>>
 ** <<creating-the-kubernetes-secret-for-external-endpoint,Creating the Kubernetes secret for an external endpoint>>
 ** <<adding-user-workload-metrics,Adding user workload metrics>>
 ** <<updating-the-multiclusterobservability-cr,Updating the multiclusterobservability CR>>
 ** <<viewing-the-status-of-metrics-export,Viewing the status of metric export>>
-* <<adding-advanced-config,Adding _advanced_ configuration>>
-* <<updating-replicas,Updating the multiclusterobservability CR replicas from the console>>
-* <<forward-alerts,Forwarding alerts>>
```
Comment (dockerymick) [inline observability/customize_observability.adoc:19]: @marcolan018 I moved Forwarding alerts to be after the Configuring AlertManager section. Can you confirm if the doc info is still accurate with the section order change?

Comment (dockerymick) [reply observability/customize_observability.adoc:19]: @subbarao-meduri ^

**Review by marcolan018 — APPROVED** [review]
lgtm

File: observability/customize_observability.adoc [hunk 2]
```diff
@@ -5,25 +5,25 @@ Review the following sections to learn more about customizing, managing, and vie
...
-* <<configuring-alertmanager,Configuring AlertManager>>
-* <<adding-custom-metrics, Adding custom metrics>>
-* <<removing-default-metrics,Removing default metrics>>
-* <<export-metrics-to-external-endpoints,Export metrics to external endpoints>>
+- <<creating-custom-rules,Creating custom rules>>
+- <<configuring-alertmanager,Configuring AlertManager>>
+- <<forward-alerts,Forwarding alerts>>
+- <<adding-custom-metrics, Adding custom metrics>>
+- <<removing-default-metrics,Removing default metrics>>
+- <<export-metrics-to-external-endpoints,Export metrics to external endpoints>>
```
Comment (swopebe) [inline observability/customize_observability.adoc]: Can we make sure the verbs are all parallel here in this list of sections?

File: observability/customize_observability.adoc [hunk 3]
```diff
@@ -5,25 +5,25 @@ Review the following sections to learn more about customizing, managing, and vie
...
+- <<adding-advanced-config,Adding _advanced_ configuration>>
+- <<updating-replicas,Updating the multiclusterobservability CR replicas from the console>>
+- <<customizing-route-cert,Customizing route certification>>
+- <<customizing-certificates-object-store,Customizing certificates for accessing the object store>>
+- <<viewing-and-exploring-data,Viewing and exploring data>>
 ** <<viewing-etcd-grafana,Viewing the etcd table>>
 ** <<viewing-cluster-fleet-service-level-overview-on-k8s-api-server-grafana,Viewing the cluster fleet service-level overview for the Kubernetes API server dashboard>>
 ** <<viewing-cluster-service-level-overview-on-k8s-api-server-grafana,Viewing the cluster service-level overview for the Kubernetes API server dashboard>>
-* <<disable-observability,Disable observability>>
+- <<disable-observability,Disable observability>>
```
Comment (swopebe) [inline observability/customize_observability.adoc]: same thing here with Disable

**Review by swopebe — CHANGES_REQUESTED** [review]
couple comments regarding the anchors/sections.

**Review by swopebe — APPROVED** [review]

**Comment by dockerymick** [comment]
Thanks for yall reviews and approvals! Merging this PR

**Pull request merged**